tools/device_file_generator/avr_io.py

```diff
@@ -1281,12 +1281,12 @@
 		[
 			{ "port": "B", "id": "2", "int": "0" },
 		],
-		"spi":
-		[
-			{ "port": "B", "id": "1", "name": "miso", "dir": "in" },
-			{ "port": "B", "id": "2", "name": "mosi", "dir": "out" },
-			{ "port": "B", "id": "3", "name": "sck", "dir": "out" }
-		],
+		#"spi":
+		#[
+		#	{ "port": "B", "id": "1", "name": "miso", "dir": "in" },
+		#	{ "port": "B", "id": "2", "name": "mosi", "dir": "out" },
+		#	{ "port": "B", "id": "3", "name": "sck", "dir": "out" }
+		#],
 		"usi":
 		[
 			{ "port": "B", "id": "2", "name": "usck", "dir": "out" },
```

Comment:
Btw, this data is available as ugly excel sheets:
https://dl.dropboxusercontent.com/u/44769046/ATtiny_pinout.ods
https://dl.dropboxusercontent.com/u/44769046/ATmega_pinout.ods